flake.nix

```diff
@@ -0,0 +1,196 @@
+                  description = "The telegram token fetched from BotFather.";
+                };
+                IMAGE_CHANNEL_ID = lib.mkOption {
+                  type = lib.types.int;
+                  description = "The ID of the room to send the menu images to.";
+                };
+                TEXT_CHANNEL_ID = lib.mkOption {
+                  type = lib.types.int;
+                  description = "The ID of the room to send the menu text to.";
+                };
+                LOGGER_CHAT_ID = lib.mkOption {
+                  type = lib.types.int;
+                  description = "The ID of the room to send errors/admin messages to.";
+                };
+                SHARE_TIME_HOUR = lib.mkOption {
+                  type = lib.types.ints.between 0 23;
+                  default = 9;
+                  description = "The hour of the time for sharing the menu. Defaults to 9.";
+                };
+                SHARE_TIME_MINUTE = lib.mkOption {
+                  type = lib.types.ints.between 0 59;
+                  default = 15;
+                  description = "The minute of the time for sharing the menu. Defaults to 15.";
+                };
+                UPDATE_DB_TIME_HOUR = lib.mkOption {
+                  type = lib.types.ints.between 0 23;
+                  default = 15;
+                  description = "The hour of the time for updating the database. Defaults to 15.";
+                };
+                UPDATE_DB_TIME_MINUTE = lib.mkOption {
+                  type = lib.types.ints.between 0 59;
+                  default = 0;
+                  description = "The minute of the time for updating the database. Defaults to 0.";
+                };
+                WEBHOOK_CONNECTED = lib.mkOption {
+                  type = lib.types.bool;
+                  default = cfg.hostname != null;
+                  description = "Whether to use webhook instead of polling for messages. Defaults to true if `services.hu-cafeteria-bot.hostname` is non-null.";
+                };
+                PORT = lib.mkOption {
+                  type = lib.types.port;
+                  default = 48879;
+                  description = "Port to listen the webhook on. Defaults to 48879.";
+                };
+                WEBHOOK_URL = lib.mkOption {
+                  type = lib.types.str;
+                  default = "https://${cfg.hostname}/${cfg.settings.TELEGRAM_API_KEY}";
+                  description = "The URL for the webhook. Defaults to hostname + api key.";
+                };
+                BACKGROUND_COLORS = lib.mkOption {
+                  type = lib.types.listOf lib.types.str;
+                  default = [ "#C9D6DF" "#F8F3D4" "#FFE2E2" "#E7D4B5" "#AEDEFC" "#EAFFD0" "#FFD3B4" "#BAC7A7" "#95E1D3" "#FCE38A" "#FFB4B4" ];
+                  description = "The list of background colors to be used for menu images. Defaults to the upstream-provided colors.";
+                };
+            };
```

Comment:
I wonder what are these description texts for? Can we set these values manually from cli?